### ISSUE TITLE
Circle Tour

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#393]
```
Application Version: SMV 5.2
SVN Revision Number: 2039
Compile Date: Jul 17 2008
Operating System: WIN XP

Describe details of the issue below:
SMV does not offer a circle tour anymore. Is this intentionally?

```

Original issue reported on code.google.com by `Info@F-Sim.de` on 2008-07-18 12:00:37

<hr>
- _Attachment: SMV5_0_6.png<br>![SMV5_0_6.png](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-414/comment-0/SMV5_0_6.png)_
- _Attachment: SMV5_2.png<br>![SMV5_2.png](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-414/comment-0/SMV5_2.png)_

**Comment by gforney** [comment]
```
I am assigning this to Glenn, who is going on vacation next week. He will probably 
check in on this from time to time.
```

Original issue reported on code.google.com by `mcgratta` on 2008-07-18 12:32:33

**Comment by gforney** [comment]
```
fixed in smokeview 5.2.2 just posted. Report back here whether this fixes problem on
your computer.
```

Original issue reported on code.google.com by `gforney` on 2008-07-18 16:31:33

**Comment by gforney** [comment]
```
presume verified
```

Original issue reported on code.google.com by `gforney` on 2008-07-28 13:16:07
